content/news/2021/2021-09-20-associate-users-with-events-to-track-event-impact.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Associate Users With Events To Track Event Impact
+date: 2021-09-20
+draft: false
+---
+
+If you've ever used any error and event monitoring service, there has probably come a point where you are pulling out your hair trying to figure out if an issue is actually a major problem or if it's impacting just one user. [Exceptionless](https://exceptionless.com), an open-source event monitoring service, helps with this out of the box by making use of event stacks. But you can extend Exceptionless's functionality and automatically associate users with events. So, that one error that happens 15,000 times a day, maybe it's just a single user who wrote some bad code. 
+
+Let's take a look at how we can associate user with events using the new [Exceptionless JavaScript client](./2021-09-09-announcing-the-new-exceptionless-javascript-client.md).
+
+We're going to create a simple Node.js Express server, drop in Exceptionless, then track events with user details tied in. To get started, let's create a new Express project. Fire up your command line terminal and run this command: 
+
+```
+npx express-generator --no-view exceptionless-users
```

Comment:
Don't we need to globally install this first?

Reply:
Nope npx keeps us from having to globally install.

Reply:
We probably should link to `express-generator` so users know how to install it / where to get it.